src-gui/src/renderer/components/other/InlineCode.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, makeStyles } from "@material-ui/core"
+import { ReactNode } from "react"
+
+type Props = {
+    content: string,
+    onClick?: (content: string) => void,
+    endIcon?: ReactNode
+}
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        display: "inline-flex",
+        alignItems: 'center',
+        backgroundColor: theme.palette.grey[900],
+        borderRadius: theme.shape.borderRadius,
+        padding: theme.spacing(0.5)
```

Comment:
I'd increase the spacing here to theme.spacing(1) and add a another attribute `[gap](https://css-tricks.com/almanac/properties/g/gap/): theme.spacing(1)` to add spacing between text and the copy button.